docs/spec_draft.md

```diff
@@ -343,6 +344,78 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_grad
+
+### Semantics
+
+Computes gradients of `Batch Normalizing Transform` for `operand`, `scale` and
+offset, for each feature in the `feature_index` dimension, using `grad_output`
+and produces `grad_operand`, `grad_scale` and `grad_offset` tensors. Refer the
+[Batch Normalization](https://arxiv.org/abs/1502.03167) paper for the detailed
+differential equations.
+
+### Inputs
+
+| Name            | Type                                        |
+|-----------------|---------------------------------------------|
+| `operand`       | tensor of floating-point type               |
+| `scale`         | 1-dimensional tensor of floating-point type |
+| `mean`          | 1-dimensional tensor of floating-point type |
+| `variance`      | 1-dimensional tensor of floating-point type |
+| `grad_output`   | tensor of floating-point type               |
+| `epsilon`       | constant of type `f32`                      |
+| `feature_index` | constant of type `si64`                     |
+
+### Outputs
+
+| Name           | Type                                        |
+|----------------|---------------------------------------------|
+| `grad_operand` | tensor of floating-point type               |
+| `grad_scale`   | 1-dimensional tensor of floating-point type |
+| `grad_offset`  | 1-dimensional tensor of floating-point type |
+
+### Constraints
```

Comment:
Missing constraints in the spec
https://github.com/openxla/stablehlo/blob/3a89d59082b2a0bb8593931f974256594dadae3a/stablehlo/dialect/StablehloOps.td#L1567

Reply:
C6, C7